### ISSUE TITLE
This screenshit needs update ASAP

### ISSUE THREAD

**Issue by RonaldDijkstra** [DefactoSoftware/website#1848]
https://www.defacto.nl/blog/huis-voor-de-sport-groningen-coacht-co-met-capp-agile-learning/

**Comment by lennartdefacto** [comment]
Can you be more precise in what you'd like to be changed?

**Comment by RonaldDijkstra** [comment]
Look at the quality of it.

**Comment by lennartdefacto** [comment]
Agreed, this is the screenshot I've been supplied with. I've asked the related accountmanager for a better image (I don't have access to it).

**Comment by RonaldDijkstra** [comment]
Mkay

**Comment by RonaldDijkstra** [comment]
@michellusthof can you fix
